doc/target.rst

```diff
@@ -450,3 +450,112 @@ Target
     Returns the path to the extracted contents. In case of files (gzip and
     bzip2), the path to the decompressed file is returned; for archives, the
     path to the directory with the archive's contents is returned.
+
+
+Android Target
+---------------
+
+.. class:: AndroidTarget(connection_settings=None, platform=None, working_directory=None, executables_directory=None, connect=True, modules=None, load_default_modules=True, shell_prompt=DEFAULT_SHELL_PROMPT, conn_cls=AdbConnection, package_data_directory="/data/data")
+
+    :class:`AndroidTarget` is a subclass of :class:`Target` with additional features specific to a device running Android.
+
+    :param conn_cls: This is the type of connection that will be used to communicate
+        with the device. The majority of Android devices will use an ADB connection.
+
+    :param package_data_directory: This is the location of the data stored for
+        installed Android packages on the device.
+
+.. method:: Target.set_rotation(rotation)
+
+   Sets the screen orientation of the device as an integer value
+   representing the screen orientations. ``Natural : 0``, ``Rotated Left: 1``,
+   ``Inverted : 2`` and ``Rotated Right : 3``.
+
+.. method:: Target.get_rotation(rotation)
+
+   Returns an integer value representing the orientation of the devices screen.
+   ``0 : Natural``, ``1 : Rotated Left``,
+   ``2 : Inverted`` and ``3 : Rotated Right``.
+
+.. method:: Target.set_natural_rotation()
+
+   Sets the screen orientation of the device to its natural orientation.
+
+.. method:: Target.set_left_rotation()
+
+   Sets the screen orientation of the device to its 90 deg orientation.
+
+.. method:: Target.set_inverted_rotation()
+
+   Sets the screen orientation of the device to its inverted (180 deg)
+   orientation.
+
+.. method:: Target.set_right_rotation()
+
+   Sets the screen orientation of the device to its 270 deg orientation.
+
+.. method:: Target.set_auto_rotation(autorotate)
+
+   Specify whether the devices auto-rotation should be enabled.
+
+.. method:: Target.get_auto_rotation()
+
+   Returns ``True`` if the targets auto rotation is currently enabled and
+   ``False`` otherwise.
+
+.. method:: Target.set_airplane_mode(mode)
+
+   Specify whether the device should be in airplane mode.
+
+   .. note:: Requires the device to be rooted if the device is running Android 7+.
+
+.. method:: Target.get_airplane_mode()
+
+   Returns ``True`` if the target is currently in airplane mode and ``False``
+   otherwise.
+
+.. method:: Target.get_brightness()
+
+   Returns the devices current screen brightness as an integer between 0 and 255.
+
+.. method:: Target.set_brightness(value)
+
+   Sets the devices screen brightness as an integer between 0 and 255 to the
+   specified value.
+
+.. method:: Target.get_brightness()
```

Comment:
Re-arrange the entries so that the the getters immediately follow/proceed (be consistent with existing docs, if applicable) the corresponding setters; rather than listing all setters followed by all getters.